course_info/templates/course_info/partials/details.html

```diff
@@ -0,0 +1,211 @@
+{% verbatim %}
+
+<nav>
```

Comment:
The whole form displays, with empty fields, before the data loads -- it would be good to show something else until the data loads so the user doesn't think it's broken. We could 'bug it' and fix it separately from the PR, or as part of 2376, to keep the stories moving along.